# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1551289583 28800
#      Wed Feb 27 09:46:23 2019 -0800
# Node ID 59bd02077bf36a4c5e334abe40b183ba2febd8bc
# Parent  11bf2c81546985710af3b352d277b6166dcda70f
maint: reintroduce OCTAVE_API_VERSION development suffix

* RELEASE.CHECKLIST: Reintroduce "+" development suffix on OCTAVE_API_VERSION.

diff --git a/etc/RELEASE.CHECKLIST b/etc/RELEASE.CHECKLIST
--- a/etc/RELEASE.CHECKLIST
+++ b/etc/RELEASE.CHECKLIST
@@ -63,17 +63,17 @@
 <!----------------------------------------------------------------------------->
 # Localization and Internationalization
 #* Update language translation files (*.ts)
 #* Create issue report on Savannah as a centralized location for uploading files
 #* Submit call for translations for GUI strings
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # Update shared library and oct file API version numbers
-#* Increment oct file API version number (configure.ac:OCTAVE_API_VERSION)
+#* Increment oct file API version number (configure.ac:OCTAVE_API_VERSION, increment number and drop "+" suffix)
 #* Increment libtool versioning (liboctave/module.mk:%canon_reldir%_%canon_reldir%_current, libinterp/module.mk:%canon_reldir%_liboctinterp_current, libgui/module.mk:%canon_reldir%_liboctgui_current)
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # Verify build process and create release candidates
 #* Update configure.ac with new version information
 #** Update AC_INIT, OCTAVE_MAJOR_VERSION, OCTAVE_MINOR_VERSION, OCTAVE_PATCH_VERSION, OCTAVE_RELEASE_DATE
 #* Verify 'make distcheck' passes
 #* Create release candidate
@@ -97,15 +97,16 @@
 #* Savannah bug tracker version info
 #** add new release version to bug tracker
 #** remove release candidate versions from bug tracker
 #* Announce final release on Octave mailing lists and web site
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # Post-Release
 #* Update configure.ac (AC_INIT, OCTAVE_MAJOR_VERSION, OCTAVE_MINOR_VERSION, OCTAVE_PATCH_VERSION) to next release cycle
+#* Update oct file API version number (configure.ac:OCTAVE_API_VERSION, add "+" suffix)
 #* Remove all deprecated functions (either OCTAVE_DEPRECATED in C++ or scripts/deprecated for m-files) scheduled for deletion in default branch
 #* Move NEWS file to backup in etc/NEWS.X
 #* Create new NEWS file
 #: Completion Date:
 
 [[Category:Releases]]
 [[Category:Development]]
